### PR TITLE
Relap support for amp-ads

### LINKED ISSUE #7010: Intent To Implement: Relap support for amp-ads
# Overview

Support for Relap widgets will be added to `amp-ad` elements

## Comments

Relap will be submitting a PR via `advzr/amphtml` fork from `relap` branch.

### PR DESCRIPTION
**Pull request by advzr**
Closes #7010

### CODE REVIEW

**Comment by advzr** [comment]
I signed it!

**Comment by advzr** [comment]
Hi. I don't want to hurry you, but when can I expect your review? My employer is asking me about it every day.

**Comment by jridgewell** [comment]
Ping @lannka, @zhouyx

**Comment by lannka** [comment]
@advzr would you try to follow the [guide](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request) and implement the render-start API?

**Comment by advzr** [comment]
@lannka Done

File: examples/ads.amp.html [hunk 1]
```diff
@@ -1013,6 +1014,18 @@ <h2 id="purch">Purch 300x250</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="relap">Relap</h2>
+  <amp-ad width=auto height=750
```
Comment (lannka) [inline examples/ads.amp.html]: nitpick: double quote around attribute values

File: examples/ads.amp.html [hunk 2]
```diff
@@ -1013,6 +1014,18 @@ <h2 id="purch">Purch 300x250</h2>
...
+      type="relap"
+      data-token="D3UMgQWBqleq1tPW"
+      data-url="http://bigpicture.ru"
+      data-anchorid="i0xMMY1MoliiZWVl"
+      layout="fixed-height"
```
Comment (lannka) [inline examples/ads.amp.html]: nit: could you move `layout` before all the data attributes?

File: examples/ads.amp.html [hunk 3]
```diff
@@ -1013,6 +1014,18 @@ <h2 id="purch">Purch 300x250</h2>
...
+  >
```
Comment (lannka) [inline examples/ads.amp.html]: move this to previous line

File: examples/ads.amp.html (hunk 1 above)
Comment (lannka) [inline examples/ads.amp.html]: Is `relap` about content recommendation or ads? For content recommendation, `amp-embed` tag is recommended, see `taboola`

Comment (advzr) [inline examples/ads.amp.html]: `relap` is about content recommendation with ads. The amount of ads depends on the publisher. The widget can be without any ads, have some or consist of only ads.

**Pull request merged**